easybuild/easyconfigs/s/Stacks/Stacks-1.35-goolf-1.4.10.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+#
+# Full Stacks pipeline is php + mysql + command line utilies
+# This will install JUST the command line utilities
+
+easyblock = 'ConfigureMake'
+
+name = "Stacks"
+version = "1.35"
+
+homepage = 'http://creskolab.uoregon.edu/stacks/'
+description = """ Stacks is a software pipeline for building loci from short-read sequences, 
+ such as those generated on the Illumina platform. Stacks was developed to work with restriction 
+ enzyme-based data, such as RAD-seq, for the purpose of building genetic maps and conducting 
+ population genomics and phylogeography. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://creskolab.uoregon.edu/stacks/source/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('SAMtools', '0.1.19'),
+    ('google-sparsehash', '2.0.2'),
```

Comment:
@pescobar name was changed to `sparsehash` a long time ago, see #1452, please fix?